modules/ROOT/pages/api-groups-sla-tiers-faqs.adoc

```diff
@@ -0,0 +1,30 @@
+= SLA Tiers For API Groups FAQs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+*Question: What happens if an API Group is published without any SLA tiers?*
+
+*Answer:*
+As a Group Administrator, you must carefully decide whether an API Group must not have any SLA tiers. No SLA implies unlimited SLA Tiers for all the instances in the APi group that do not have an SLA based policy applied. 
```

Comment:
This text was not written by you. Please, you MUST edit Dev. input and you MUST reread your output before submitting to me.
```suggestion
As a Group Administrator, you must carefully consider whether an API Group is not allowed any SLA tiers. No SLA implies unlimited SLA tiers for all the instances in the API group that do not have an SLA-based policy applied. 
```

Reply:
Page to be removed. Done! Thanks!